packages/api/src/external/commonwell/patient-demographics.ts

```diff
@@ -0,0 +1,95 @@
+import { Patient } from "@metriport/core/domain/patient";
+import { LinkDemographics } from "@metriport/core/domain/patient-demographics";
+import { PatientNetworkLink, GenderCodes } from "@metriport/commonwell-sdk";
+import {
+  scoreLink,
+  linkHasNewDemographics,
+  patientToNormalizedCoreDemographics,
+  removeInvalidArrayValues,
+  normalizeDob,
+  normalizeGender,
+  normalizeAndStringifyNames,
+  normalizeAddress,
+  stringifyAddress,
+  normalizeTelephone,
+  normalizeEmail,
+} from "../../domain/medical/patient-demographics";
+import { CwLink } from "./cw-patient-data";
+
+type CwGenderCode = `${GenderCodes}`;
```

Comment:
It seems we don't need this, can just use `GenderCodes`?

Reply:
Done -- I forget why I had that casting before but you're right, unnecessary.